controller/features_test.go

```diff
@@ -2,21 +2,29 @@ package controller
 
 import (
 	"context"
+	jwt "github.com/dgrijalva/jwt-go"
 	"github.com/fabric8-services/fabric8-toggles-service/app/test"
 	"github.com/goadesign/goa"
+	goajwt "github.com/goadesign/goa/middleware/security/jwt"
 	"github.com/magiconair/properties/assert"
 	"testing"
 )
 
 func TestFeatures(t *testing.T) {
 	svc := goa.New("feature")
-	ctrl := NewFeaturesController(svc)
+	ctrl := NewFeaturesController(svc, nil)
 	expectedFeaturesList := buildFeaturesList(5)
 
-	t.Run("OK", func(t *testing.T) {
+	t.Run("OK with no token", func(t *testing.T) {
 		_, featuresList := test.ListFeaturesOK(t, context.Background(), svc, ctrl)
-		assert.Equal(t, 5, len(featuresList.Data))
-		assert.Equal(t, featuresList.Data[0].Attributes.Name, expectedFeaturesList.Data[0].Attributes.Name)
+		assert.Equal(t, 0, len(featuresList.Data))
+		//assert.Equal(t, featuresList.Data[0].Attributes.Name, expectedFeaturesList.Data[0].Attributes.Name)
+	})
+	t.Run("OK with jwt token", func(t *testing.T) {
+		_, featuresList := test.ListFeaturesOK(t, createValidContext(), svc, ctrl)
+		assert.Equal(t, 2, len(featuresList.Data))
```

Comment:
you should use `require` instead of `assert`, so the execution will stop if `featureList` is nil or does not have the expected size. Otherwise, the execution will continue and might result in a `panic`